ansible/roles/infra-osp-template-generate/defaults/main.yaml

```diff
@@ -8,3 +8,71 @@ opentlc_admin_pub_keys:
 all_ssh_authorized_keys: "{{ opentlc_admin_pub_keys + [user_pub_key|d()] }}"
 
 provider_network: external
+
+default_security_groups:
+  - name: BastionSG
+    description: >-
+      Bastion security group allows basic ICMP
+      and SSH adn Mosh ingress and egress to *
+    rules:
+      - name: MoshPublic
```

Comment:
We do not need this in a default. I'd suggest most people don't even know what mosh is or use it.

Reply:
We have it in default for aws, and i kind of like Mosh.

Does it hurt ?

Reply:
Per our conversation...

Does it hurt? No. Is it needed? No. These are defaults. Defaults should be needed either for functionality of a config/role or by a lab. Nothing requires this, so it should not be included going forward just because it was included in the past.

If there is a guaranteed repository for this package - or any like it - that is consistently maintained for updates, security fixes, etc. I'd be more okay with it. In the past we used EPEL, which can break things easily. Recently, we started hosting some packages from an s3 bucket, but I don't know who is ensuring that stays up to date.

Reply:
I just removed it from the default security groups.